app/decorators/service_decorator.rb

```diff
@@ -1,6 +1,6 @@
 class ServiceDecorator < MiqDecorator
   def fonticon
-    nil
+    'product product-service'
```

Comment:
@skateman this should be 'pficon pficon-service' (product-service no longer exists)